package.json

```diff
@@ -53,6 +53,7 @@
     "wait-on": "^6.0.1"
   },
   "devDependencies": {
+    "eslint": "^8.23.1",
```

Comment:
Not sure if we really need eslint here in the root repository. I think when lint script is executed it is actually calling lint in every app where we should have eslint installed already as their own dependency. Maybe I am wrong but I think this worked fine so far.

Reply:
I will remove it